MAUI/TreeMap/brush-settings.md

```diff
@@ -0,0 +1,438 @@
+---
+layout: post
+title: Brush settings in .NET MAUI TreeMap (SfTreeMap) control | Syncfusion
+description: Learn here how to apply brush settings for tree map items in Syncfusion .NET MAUI TreeMap control, enhancing the tree map visualization.
+platform: maui
+control: TreeMap (SfTreeMap)
+documentation: ug
+---
+ 
+# Leaf item brush settings in .NET MAUI TreeMap (SfTreeMap)
+
+The brush settings is used to customize the fill colors for leaf items based on ranges or values, offering four brush settings: `UniformBrushSettings,` `RangeBrushSettings,` `DesaturationBrushSettings,` and `PaletteBrushSettings.` Each setting provides unique options for defining and applying color schemes, enhancing the visualization of the tree map.
+
+## Uniform brush settings
+
+The uniform brush settings in `SfTreeMap` assigns a single color uniformly for leaf items representing data. Customize the background brush for tree map leaf items using the `Brush` property of `TreeMapUniformBrushSettings`.
```

Comment:
The uniform brush settings in `SfTreeMap` assign a single color uniformly for leaf items representing data. Customize the background brush for treemap leaf items using the `Brush` property of `TreeMapUniformBrushSettings`.

Reply:
Updated.